development/Obtain_dataset.ipynb

```diff
@@ -0,0 +1,243 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Load the dataset\n",
+    "\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To run this notebook, you will need to install: pandas, openai, transformers, plotly, matplotlib, scikit-learn, torch (transformer dep), torchvision, and scipy."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 49,
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "C:\\Users\\Lars\\AppData\\Local\\Temp\\ipykernel_13256\\3593873765.py:40: SettingWithCopyWarning: \n",
+      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
+      "Try using .loc[row_indexer,col_indexer] = value instead\n",
+      "\n",
+      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
+      "  data[\"Embeddings\"] = embeddings\n"
+     ]
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "import openai\n",
+    "\n",
+    "# Setup API-Key for OpenAi\n",
+    "openai.api_key = \"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\"\n",
+    "\n",
+    "# Define the embedding model which is used\n",
+    "embedding_model = \"text-embedding-ada-002\"\n",
+    "\n",
+    "# Retrieve the data from the database\n",
+    "input_datapath = pd.read_sql_query(\"SELECT * FROM 'mortgage complaints'\", \"sqlite:///StaterData.db\")\n",
+    "\n",
+    "# Limit test size due to performance issues\n",
+    "data = input_datapath.loc[:100]\n",
+    "\n",
+    "# Extract the relevant columns and put into a list\n",
+    "texts = data[\"Consumer complaint narrative\"].tolist()\n",
+    "\n",
+    "# Initialize a list to store the embeddings\n",
+    "embeddings = []\n",
+    "\n",
+    "# Generate embeddings for the texts\n",
+    "for text in texts:\n",
+    "    response = openai.Completion.create(\n",
```

Comment:
Why completion.create, function is used to complete an sentence, not to make embeddings

Reply:
I agree with Bram. 

This links shows a simple example with openai.Completion.create:
https://platform.openai.com/examples/default-openai-api/

Reply:
not sure how to make the changes

Reply:
I could make these changes but there's no way I can test at the moment due to exceeding my quota.

Reply:
> I could make these changes but there's no way I can test at the moment due to exceeding my quota.

It's okay! I understand that you can't test this as you have exceeded the quota. Today, I have worked with the embeddings.

Reply:
I will resolve this conversation as this will be handled in STAT-11 branch